Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -10889,6 +10889,29 @@ def is_scheduled_command_retry():
     return True if sm.get('is_polling', False) else False
 
 
+def replace_spaces_in_certificate(private_key):
+    """
+    This function is used in case the private key is in the wrong format of one line with spaces instead of multiple lines.
```

Comment:
```suggestion
    This function is used in case credential from type: 9 is in the wrong format of one line with spaces instead of multiple lines.
```